06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/README.md

```diff
@@ -4,4 +4,4 @@
 - **With**: *independently*
 
 #### Targeted Learning Outcomes
-- [Software Development Environments](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/35%20-%20Software%20Development%20Environments)
```

Comment:
I must have edited it. The output for the task is in the 'output' directory in the 'setting-up-software-development-environments-(deployment)' directory.